tests/integration/targets/ec2_lc/tasks/main.yml

```diff
@@ -0,0 +1,177 @@
+- name: run ec2_lc tests
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  collections:
+    - amazon.aws
+
+  block:
+
+    - name: set up environment for testing.
+      include_tasks: env_setup.yml
+
+# Create lc 1
+    - name: Create launch configuration
+      community.aws.ec2_lc:
+        name: '{{ resource_prefix }}-lc1'
+        image_id: '{{ ec2_ami_id }}'
+        assign_public_ip: yes
+        instance_type: '{{ ec2_instance_type }}'
+        security_groups: '{{ sg.group_id }}'
+        volumes:
+          - device_name: /dev/xvda
+            volume_size: 10
+            volume_type: gp2
+            delete_on_termination: true
+      register: lc_1_create
+
+    - assert:
+        that:
+          - lc_1_create is changed
+          - '"autoscaling:CreateLaunchConfiguration" in lc_1_create.resource_actions'
```

Comment:
Could you also please add more assertions statements (to check whether the result is correct), as I did here https://github.com/ansible-collections/community.aws/pull/687/files#diff-e273004cf14f6d5825194f00d8ad0e4559b7640000f0cfc967f16f781ceecf59R46? I suggest to do for all the tasks where it makes sense to have them added.